src/cloud/benchflow/data-analyses-scheduler/main.go

```diff
@@ -58,7 +58,7 @@ func main() {
 	NAnalyserWorkers = viper.GetInt("analyser_workers")
 	
 	// Getting dependencies configuration and unmarshaling in defined structures
-	dat, err := ioutil.ReadFile("./configuration/analyser-scripts-configuration.yml")
+	dat, err := ioutil.ReadFile("./configuration/analysers.scheduler.configuration.yml")
```

Comment:
You have `analysers_configuration_file_name` configuration variable that can be used here